editor/components/AppNavBar.tsx

```diff
@@ -46,8 +46,11 @@ export default function AppNavBar({ user, logout }: NavBarProps) {
           </Navbar.Brand>
           <EnvironmentBadge />
         </Col>
-        <Col className="d-none d-md-flex justify-content-end align-items-center">
-          <NavBarSearch />
+        <Col className="d-flex justify-content-end align-items-center">
+          {/* Hide navbar  search on xs and sm */}
+          <div className="d-none d-md-flex">
+            <NavBarSearch />
+          </div>
```

Comment:
Fixes a bug I introduced in the last release in which the navbar dropdown menu is hidden on `xs` and `sm`. 

We have https://github.com/cityofaustin/atd-data-tech/issues/24283 to deal with the mobile nav